frontend/src/components/formsContainer/FormsContainer.js

```diff
@@ -0,0 +1,20 @@
+import CharacterForm from '../forms/CharacterForm.js'
+import StyleForm from '../forms/StyleForm.js'
+import GenreForm from '../forms/GenreForm.js'
+import LocationForm from '../forms/LocationForm.js'
+import "./FormsContainer.css";
+
```

Comment:
potentially we can require a file here that collects all the dropdown menu options (or we can hardcode them inside this formcontainer, as long as the data is coming from a singular source it shoulde be fine)

keep in mind that if you decide to go with the former route of having all options in one single file, you cannot move it outside the frontend folder (react does not send anything to the client outside that folder)